exec/cnex/CMakeLists.txt

```diff
@@ -12,10 +16,17 @@ add_executable(cnex
     number.c
     stack.c
     util.c
+    CNEX_RTL
```

Comment:
This needs to be `${CNEX_RTL}` to expand the variable. (I think older versions of cmake would automatically expand it, but now you have to do it explicitly.)

Reply:
Fixed.